src/components/sbb-journey-header/sbb-journey-header.scss

```
@@ -5,58 +5,45 @@
 @include sbb.host-component-properties;
 
 :host {
-  --_icon-spacing-inset: var(--sbb-spacing-fixed-1x);
   --sbb-title-text-color-normal: var(--sbb-color-charcoal-default);
+  --sbb-journey-header-gap: var(--sbb-spacing-fixed-1x);
 }
 
-.journey-header[id] {
+:host([negative]:not([negative='false'])) {
+  @include sbb.title--negative;
+}
+
+:host([id]) {
   @include sbb.scroll-margin-block-start;
 }
 
-.journey-header {
+sbb-title {
+  --sbb-title-margin-block-start: 0;
+  --sbb-title-margin-block-end: 0;
```

It's easier like that and also consistent with other cases in code base.
```suggestion
  // Overwrite sbb-title default margin
  margin: 0;
```